### ISSUE TITLE
Add alias to .bashrc for more succinct command

### ISSUE THREAD

**Issue by BrendanNeufeld** [Jam3/nyg-ui-generator#9]
We should be able to do this through a predefined bash command. We should also come up with the best possible name. I have been just using `ui` as an alias for `nyg nyg-ui-generator`.

**Comment by nascherman** [comment]
It would be possible to set some alias as part of a postinstall script, including maybe some for quickly creating modules of a certain type.

**Comment by BrendanNeufeld** [comment]
Yes! That is exactly what I was getting at. Definitely an alias for ui-generator and module-generator. Let me know what else you have in mind.

**Comment by nascherman** [comment]
those two sound good. i'll try and think of some more aliases.
